src/app/framework-modules/elements/elements.ts

```diff
@@ -224,15 +220,19 @@ import { CtHorizontalBarChartComponent } from './ct-horizontal-barchart/componen
         CtNumberCardComponent,
         CtLinearGaugeComponent,
         CtHorizontalBarChartComponent,
+        CTFieldGenerator,
+        CTTextareaComponent,
         /**
          * Miscellaneous Exports
          */
         ReactiveFormsModule,
         CommonModule,
         FormsModule,
```

Comment:
Javadoc can pick up on this?  This should really be a `/**` and not just `//` or `/*`?